ctest-next/templates/test.rs

```diff
@@ -0,0 +1,111 @@
+/* This file was autogenerated by ctest; do not modify directly */
+{#- ↑ Doesn't apply here, this is the template! +#}
+
+/// As this file is sometimes built using rustc, crate level attributes
+/// are not allowed at the top-level, so we hack around this by keeping it
+/// inside of a module.
+mod generated_tests {
+    #![allow(non_snake_case)]
+    #![deny(improper_ctypes_definitions)]
+    use super::*;
+
+    use std::mem;
+    use std::sync::atomic::{AtomicBool, AtomicUsize, Ordering};
+    use std::fmt::{Debug, LowerHex};
+    use std::ptr;
+    use std::slice;
+    use std::ffi::CStr;
```

Comment:
```suggestion
    use std::ffi::CStr;
    use std::fmt::{Debug, LowerHex};
    use std::sync::atomic::{AtomicBool, AtomicUsize, Ordering};
    use std::{mem, ptr, slice};

    use super::*;
```

Making this match rustfmt